crates/sui-config/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub struct ValidatorConfig {
     network_address: Multiaddr,
     metrics_address: Multiaddr,
 
-    consensus_config: ConsensuseConfig,
+    pub consensus_config: ConsensuseConfig,
```

Comment:
these don't need to be made pub as there are accessors implemented below for that

Reply:
Ideally we need a constructor for narwhal configs, but I figured this could be made in a separate PR (the config structure is pretty complex)

Reply:
Got it. Yeah this is something that we can fix later (enabling configuration generation with a base template)